repl/src/main/scala/com/cloudera/livy/repl/Main.scala

```diff
@@ -22,14 +22,14 @@ import java.util.concurrent.TimeUnit
 import javax.servlet.ServletContext
 
 import scala.annotation.tailrec
-import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
 
-import dispatch._
+import com.ning.http.client.AsyncHttpClient
 import org.json4s.{DefaultFormats, Formats}
 import org.json4s.jackson.Serialization.write
-import org.scalatra.LifeCycle
 import org.scalatra.servlet.ScalatraListener
+import org.scalatra.LifeCycle
```

Comment:
nit: this was correct in the previous location; the style checker doesn't catch that because of a bug (that I've fixed but hasn't been released yet).